src/essentia/utils/extractor_freesound/FreesoundTonalDescriptors.cpp

```diff
@@ -87,97 +89,35 @@ void FreesoundTonalDescriptors ::createNetwork(SourceBase& source, Pool& pool) {
-                                   "slope", 0.6,
-                                   "usePolyphony", true,
-                                   "useThreeChords", true);
-
   spec->output("spectrum") >> hpcp_peaks->input("spectrum");
   hpcp_peaks->output("frequencies") >> hpcp_key->input("frequencies");
   hpcp_peaks->output("magnitudes")  >> hpcp_key->input("magnitudes");
-
   hpcp_key->output("hpcp")     >> PC(pool, nameSpace + "hpcp");
-  
-  hpcp_key->output("hpcp")     >> skey_temperley->input("pcp");
-  hpcp_key->output("hpcp")     >> skey_krumhansl->input("pcp");
-  hpcp_key->output("hpcp")     >> skey_edma->input("pcp");
-
-  skey_temperley->output("key")          >> PC(pool, nameSpace + "key_temperley.key");
-  skey_temperley->output("scale")        >> PC(pool, nameSpace + "key_temperley.scale");
-  skey_temperley->output("strength")     >> PC(pool, nameSpace + "key_temperley.strength");
-
-  skey_krumhansl->output("key")          >> PC(pool, nameSpace + "key_krumhansl.key");
-  skey_krumhansl->output("scale")        >> PC(pool, nameSpace + "key_krumhansl.scale");
-  skey_krumhansl->output("strength")     >> PC(pool, nameSpace + "key_krumhansl.strength");
-
-  skey_edma->output("key")          >> PC(pool, nameSpace + "key_edma.key");
-  skey_edma->output("scale")        >> PC(pool, nameSpace + "key_edma.scale");
-  skey_edma->output("strength")     >> PC(pool, nameSpace + "key_edma.strength");
-
-  // Compute chords
-  // TODO review these parameters to improve chords detection. Keeping old code for now
-  Algorithm* hpcp_chord = factory.create("HPCP",
-                                         "size", 36,
-                                         "referenceFrequency", tuningFreq,
-                                         "harmonics", 8,
-                                         "bandPreset", true,
-                                         "minFrequency", 40.0,
-                                         "maxFrequency", 5000.0,
-                                         "bandSplitFrequency", 500.0,
-                                         "weightType", "cosine",
-                                         "nonLinear", true,
-                                         "windowSize", 0.5);
-  peaks->output("frequencies") >> hpcp_chord->input("frequencies");
-  peaks->output("magnitudes") >> hpcp_chord->input("magnitudes");
 
-  Algorithm* schord = factory.create("ChordsDetection");
-  hpcp_chord->output("hpcp") >> schord->input("pcp");
-  schord->output("chords") >> PC(pool, nameSpace + "chords_progression");
-  schord->output("strength") >> PC(pool, nameSpace + "chords_strength");
-  
-  Algorithm* schords_desc = factory.create("ChordsDescriptors");
-  schord->output("chords") >> schords_desc->input("chords");
-  skey_temperley->output("key") >> schords_desc->input("key");
-  skey_temperley->output("scale") >> schords_desc->input("scale");
+  // Compute key
+  Algorithm* key_extractor = factory.create("KeyExtractor",
```

Comment:
@ffont This is using the default `profileType` key profile from KeyExtractor. We can be more explicit and manually configure it here, if you need to ensure using a specific key profile.

Reply:
No need of a specific profile, let's use the default.

Reply:
The profiles we previously used are: `temperley`, `krumhansl`, `edma`. KeyExtractor defaults to `bgate`.